test/Cpp/src/DoublePort.lf

```diff
@@ -0,0 +1,62 @@
+/** 
+ * Test the case where two upstream reactors
+ * pass messages to a downstream reactor on two
+ * different ports. One message carries
+ * a microstep delay relative to the other.
+ * 
+ * @author Maiko Brants 
+ */
+target Cpp {
+    timeout: 900 msec,
+    fast: true
+};
+
+import Count from "lib/Count.lf";
+
+reactor CountMicrostep {
+    state count:int(1);
+    output out:int;
+    logical action act:int;
+    timer t(0, 1 sec);
+    reaction(t) -> act {=
+        act.schedule( count);
+        count++;
+    =}
+
+    reaction(act) -> out {=
+        out.set(act.get());
+    =}
+}
+
+reactor Print {
+    input in:int;
+    input in2:int;
+    reaction(in, in2) {=
+        auto elapsed_time = get_elapsed_logical_time();
+        if(in.is_present()){
+            reactor::log::Info() << "At tag (" << elapsed_time << ", " << get_logical_time() 
```

Comment:
`get_logical_time()` will return a time point on the logical time axis. This is not the microstep. Unfortunately, there is currently no convenient API for getting a tag in C++ (see lf-lang/reactor-cpp#3). Currently, you would need to use `environment()->logical_time().micro_step()` to get the microstep of the current tag.